Packages/com.unity.inputsystem/Documentation~/Events.md

```diff
@@ -12,9 +10,10 @@
     * [Creating events](#creating-events)
     * [Capturing events](#capturing-events)
 
-
 The Input System is event-driven. All input is delivered as events, and you can generate custom input by injecting events. You can also observe all source input by listening in on the events flowing through the system.
 
+>__Note__: Events are an advanced, mostly internal feature of the Input System. You don't need to understand them to use the Input System. Knowledge of the event system is mostly useful if you want to support custom Devices, or change the behavior of existing Devices.
```

Comment:
As discussed on Slack, I think it's OK to remove the second sentence from this note ("You don't need to understand...").